js/renderer.ts

```diff
@@ -830,6 +830,26 @@ export abstract class ReactUiBase extends Component implements ReactComp {
     @property.int(100)
     height = 100;
 
+    /**
+     * If set to 'true', the UI will be scaled automatically.
+     * If set to 'false', 'manualHeight' is used instead, and the UI will scale proportionally to screen's height,
+     * always remaining the same relative size regardless of the resolution changes.
+     */
+    @property.bool(true)
+    automatic = true;
```

Comment:
Maybe we should make this a enum `scalingType` or something with `scale to height`, `scale to width`, `absolute`?